plugins/languages/rustaceanvim.nix

```diff
@@ -0,0 +1,281 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+with lib; let
+  cfg = config.plugins.rustaceanvim;
+in {
+  meta.maintainers = [maintainers.GaetanLepage];
+
+  options.plugins.rustaceanvim =
+    helpers.extraOptionsOptions
+    // {
+      enable = mkEnableOption "rustaceanvim";
+
+      package = helpers.mkPackageOption "rustaceanvim" pkgs.vimPlugins.rustaceanvim;
+
+      rustAnalyzerPackage = mkOption {
+        type = with types; nullOr package;
+        default = pkgs.rust-analyzer;
+        description = ''
+          Which package to use for `rust-analyzer`.
+          Set to `null` to disable its automatic installation.
+        '';
+        example = null;
+      };
+
+      tools = {
+        executor =
+          helpers.defaultNullOpts.mkNullable
+          (
+            with helpers.nixvimTypes;
+              maybeRaw
```

Comment:
I'd suggest either strLuaFn (enum ....) to help in the documentation

Reply:
Done !